.travis.yml

```diff
@@ -14,7 +14,25 @@ before_install:
   - sudo add-apt-repository "deb http://archive.ubuntu.com/ubuntu cosmic universe"
   - sudo apt update
   - sudo apt install libopenbabel-dev
+
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - hash -r
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update -q conda
+  - conda info -a
+  - if [[ "$PYTHON_VER" == "3.6" ]]; then
```

Comment:
Can you try removing this branch (which just pins a specific version of NumPy)? It's actually the other one that runs for 2.7, which seems to work fine, so it's probably unnecessary.

Reply:
Done.